inference-engine/samples/ngraph_function_creation_sample/main.cpp

```diff
@@ -272,7 +272,8 @@ int main(int argc, char* argv[]) {
         }
 
         /** Setting batch size using image count **/
-        size_t batchSize = 1;
+        network.setBatchSize(imagesData.size());
```

Comment:
I propose to don't use `setBatchSize()` method in samples.
Could you use reshape method instead?